api/v2.0/swagger.yaml

```diff
@@ -6738,11 +6758,47 @@ definitions:
         type: string
         description: Version of the scanner adapter
         example: "v0.9.1"
+  ScanRequestType:
+    type: object
+    properties:
+      scan_type: 
+        type: string
+        description: 'The scan type for scan requst, it could be vulnerability or sbom'
```

Comment:
The scan type for the scan request. Two options are currently supported, `vulnerability? and `sbom`.